cmd/kpod/ps.go

```diff
@@ -585,18 +588,19 @@ func matchesFilter(ctrData *libkpod.ContainerData, params *FilterParamsPS) bool
 }
 
 // GetContainersMatchingFilter returns a slice of all the containers that match the provided filter parameters
-func getContainersMatchingFilter(containers []*oci.Container, filter *FilterParamsPS, server *libkpod.ContainerServer) []*libkpod.ContainerData {
+func getContainersMatchingFilter(containers []*oci.Container, filter *FilterParamsPS, server *libkpod.ContainerServer) ([]*libkpod.ContainerData, error) {
 	var filteredCtrs []*libkpod.ContainerData
 	for _, ctr := range containers {
 		ctrData, err := server.GetContainerData(ctr.ID(), true)
 		if err != nil {
+			return nil, err
```

Comment:
I think you want to flip this and the next line.

Reply:
backed out this change per previous comments